images/scorecard-test-kuttl/cmd/test-kuttl/main.go

```diff
@@ -175,3 +188,33 @@ type Testsuites struct {
 	// Testsuite is a collection of test suites
 	Testsuite []*Testsuite `xml:"testsuite" json:"testsuite,omitempty"`
 }
+
+func getKuttlLogs() string {
+	stderrFile, err := os.Open("/tmp/kuttl.stderr")
+	if err != nil {
+		printErrorStatus(fmt.Errorf("could not open kuttl stderr file %v", err))
+		return err.Error()
+	}
+	defer stderrFile.Close()
+
+	var byteValue []byte
+	byteValue, err = ioutil.ReadAll(stderrFile)
+	if err != nil {
+		printErrorStatus(fmt.Errorf("could not read kuttl stderr file %v", err))
+		return err.Error()
+	}
```

Comment:
These two calls could be simplified to one with [`ioutil.ReadFile`](https://golang.org/pkg/io/ioutil/#ReadFile).

Ditto for `kuttl.stdout`

Reply:
updated the code to use ioutil.ReadFile, good catch.